libs/zsdl/src/sdl2.zig

```diff
@@ -1450,6 +1450,17 @@ pub const GameController = opaque {
     extern fn SDL_GameControllerGetButton(controller: *GameController, button: c_int) u8;
 };
 
+//--------------------------------------------------------------------------------------------------
+//
+// Cursor
+//
+//--------------------------------------------------------------------------------------------------
+
```

Comment:
These section comments correspond to the categories defined by the SDL docs (see https://wiki.libsdl.org/SDL2/APIByCategory). Please move `showCursor` under the "Mouse Support" section comment and delete this one